vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/WaitSteps.java

```diff
@@ -83,7 +83,7 @@ public void waitTillAlertAppears()
      * a potentially damaging operation.
      * </p>
      */
-    @When("I wait until an alert disappears")
+    @When("I wait until alert disappears")
     public void waitTillAlertDisappears()
     {
         waitActions.wait(getWebDriver(), not(alertIsPresent()));
```

Comment:
how does assertion message look like?

Reply:
> how does assertion message look like?

![image](https://github.com/vividus-framework/vividus/assets/58571912/f775861a-53c0-4ffc-a41e-5335dc56006c)
Expected condition failed: waiting for condition to not be valid: alert to be present

Reply:
Assertion message from the new step:
`Expected condition failed: waiting for alert disappearance from the page. Passed: false (tried for 10 second(s) with 2000 milliseconds interval)`